docs/generatedcode/python.md

```diff
@@ -6,3 +6,147 @@ type: markdown
+
+## Illustrative Example
+
+Let's look at the following `FortuneTeller` proto service:
+
+```proto
+service FortuneTeller {
+  // Returns the horoscope and zodiac sign for the given month and day.
+  rpc TellFortune(HoroscopeRequest) returns (HoroscopeResponse) {
+    // errors: invalid month or day, fortune unavailable
+  }
+
+  // Replaces the fortune for the given zodiac sign with the provided one.
+  rpc SuggestFortune(SuggestionRequest) returns (SuggestionResponse) {
+    // errors: invalid zodiac sign
+  }
+}
+```
+
+gRPC `protoc` plugin will synthesize code elements along the lines
+of what follows in the corresponding `_pb2_grpc.py` file:
+
+
+```python
+import grpc
+
+import fortune_pb2
+
+class FortuneTellerStub(object):
+
+  def __init__(self, channel):
+    """Constructor.
+
+    Args:
+      channel: A grpc.Channel.
+    """
+    self.TellFortune = channel.unary_unary(
+        '/example.FortuneTeller/TellFortune',
+        request_serializer=fortune_pb2.HoroscopeRequest.SerializeToString,
+        response_deserializer=fortune_pb2.HoroscopeResponse.FromString,
+        )
+    self.SuggestFortune = channel.unary_unary(
+        '/example.FortuneTeller/SuggestFortune',
+        request_serializer=fortune_pb2.SuggestionRequest.SerializeToString,
+        response_deserializer=fortune_pb2.SuggestionResponse.FromString,
+        )
+
+
+class FortuneTellerServicer(object):
+
+  def TellFortune(self, request, context):
+    """Returns the horoscope and zodiac sign for the given month and day.
+    errors: invalid month or day, fortune unavailable
+    """
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+  def SuggestFortune(self, request, context):
+    """Replaces the fortune for the given zodiac sign with the provided
+one.
+    errors: invalid zodiac sign
+    """
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+
+def add_FortuneTellerServicer_to_server(servicer, server):
+  rpc_method_handlers = {
+      'TellFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.TellFortune,
+          request_deserializer=fortune_pb2.HoroscopeRequest.FromString,
+          response_serializer=fortune_pb2.HoroscopeResponse.SerializeToString,
+      ),
+      'SuggestFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.SuggestFortune,
+          request_deserializer=fortune_pb2.SuggestionRequest.FromString,
+          response_serializer=fortune_pb2.SuggestionResponse.SerializeToString,
+      ),
+  }
+  generic_handler = grpc.method_handlers_generic_handler(
+      'example.FortuneTeller', rpc_method_handlers)
+  server.add_generic_rpc_handlers((generic_handler,))
+```
+
+## Code Elements
+
+The gRPC code generator starts by importing `grpc` package and the plain `_pb2`
```

Comment:
"The module generated by the gRPC code generator starts by ..."? Or "The generated code starts by ..."? The gRPC code generator is not in operation after the generated module has been generated.